docker-entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -eu
+
+DATASTORE_PATH="${DATASTORE_PATH:-/datastore}"
+
+# If the first argument looks like a flag, assume we want to run changedetection
+if [ "${1:0:1}" = '-' ]; then
+    set -- python /app/changedetection.py "$@"
+fi
+
+# If we're running as root, by default make sure process uid/gid
+# and datadir permissions are correct. This can be skipped by setting
+# KEEP_PRIVILEGES to something non-empty.
+if [ "$(id -u)" = '0' -a -z "${KEEP_PRIVILEGES:-}" ]; then
```

Comment:
Maybe all this should only run linux?

The commands groupmod, usermod, and gosu are linux only

Reply:
The container is always a Linux one.

Reply:
ahhh ofcourse :) thanks